src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_configs/launch_analysis.cljs

```diff
@@ -9,12 +9,37 @@
     [org.broadinstitute.firecloud-ui.common.style :as style]
     [org.broadinstitute.firecloud-ui.common.table :as table]
     [org.broadinstitute.firecloud-ui.endpoints :as endpoints]
+    [org.broadinstitute.firecloud-ui.utils :as utils]
     ))
 
 
 (defn- entity->id [entity]
   {:type (entity "entityType") :name (entity "name")})
 
+;; GAWB-662
+;; show Queued first, then Launching, and collapse all remaining states into Active
+(defn queue-status-remap [queue-status]
```

Comment:
Could you make this insert a 0 if there are no workflows in a given state? I would prefer to see:

```
0 Queued
0 Launching
3 Active
```

instead of

```
  Queued
  Launching
3 Active
```

You could perhaps do this in queue-status-table-row, but I think it would be best to do it here, as part of the model.